vs/CheckDeps.bat

```diff
@@ -0,0 +1,288 @@
+@echo off

```

Comment:
Do you mind adding a top-level comment to this file on what it does, and in what context it is meant to be used?